index.html

```diff
@@ -193,7 +189,10 @@ <h6>University Of Studies, Poland, Cracow</h6>
             FOOTER
         </div>
     </footer>
-    <script src="./js/main.js" type="module"></script>
```

Comment:
why this was deleted? :(

Reply:
Its not deleted it just changed place?

Reply:
buvo <script src="./js/main.js" type="module"></script> 
o dabar <script src="./js/main.js"></script>
kodel type="module" istrinei? nes as kai dariau my service block su js, mano js file'ams reikalingas tas type module